src/smoke-tests/smoke-tests.js

```diff
@@ -0,0 +1,121 @@
+const { spawnSync } = require('child_process');
+const crypto = require('crypto');
+const fs = require('fs-extra');
+const os = require('os');
+const path = require('path');
+
+require('should');
+const fetch = require('node-fetch');
+
+describe('smoke tests - going to take some time', () => {
+  const context = {
+    package: {
+      filename: null,
+      version: null,
+      path: null
+    },
+    workdir: null,
+    cliBin: null
+  };
+
+  before(() => {
+    if (process.env.DEPLOY_KEY) {
```

Comment:
`DEPLOY_KEY` is already configured on Travis and it associates to a test account I created on Zapier.com.

Reply:
cool! is that anywhere in 1P or somewhere others will be able to access it?

Reply:
Yes! The account is cli-test@zapier.rodeo. The deploy key can be found there in the Form Details.